metadata/ApiCore/Testing/Mocks.php

```diff
@@ -1,6 +1,6 @@
 <?php
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
-# source: src/ApiCore/Testing/mocks.proto
```

Comment:
Couldn't get this change to go away. I generated with the following commands:

```sh
// in root of gax-php repo
$ protoc --php_out=src/ --proto_path=src/ src/ApiCore/Testing/mocks.proto
```

Reply:
Odd. I just played around a bit and saw the same thing.